### PR TITLE
Add launch URL fetch button

### PR DESCRIPTION
**Pull request by wowkalucky**
speaks for itself

### CODE REVIEW

File: bridge_adaptivity/module/templates/module/base.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
   <title>{% block bootstrap3_title %}Bridge DASHBOARD{% endblock %}</title>
   {% bootstrap_css %}
   <link rel="stylesheet" href="{% static 'module/css/module.css' %}"/>
+  <script src="{% static 'module/js/vendor/clipboard.min.js' %}"></script>
```
Comment (wowkalucky) [inline bridge_adaptivity/module/templates/module/base.html]: It is possible to use one of remote [storages](https://github.com/zenorocha/clipboard.js/wiki/CDN-Providers), but I experienced a page loading delay ones, so I decided include script to static...

File: bridge_adaptivity/module/views.py [hunk 2]
```diff
@@ -16,10 +16,10 @@
 from api.backends.openedx import get_available_courses, get_content_provider
 from bridge_lti import outcomes
 from bridge_lti.outcomes import calculate_grade
+from module import utils
 from module.forms import ActivityForm
 from module.mixins import CollectionIdToContextMixin, LtiSessionMixin
 from module.models import Collection, Activity, SequenceItem, Log, Sequence
-from module import utils
```
Comment (wowkalucky) [inline bridge_adaptivity/module/views.py]: Made imports `true`

Comment (idegtiarov) [reply bridge_adaptivity/module/views.py]: 👍

File: bridge_adaptivity/module/templates/module/base.html (hunk 1 above)
Comment (idegtiarov) [reply bridge_adaptivity/module/templates/module/base.html]: That is really not needed, CDN link would be fair enough. After page loading js lib will be cached.

**Pull request merged**